src/Domain/Exception/InvalidEntityClassException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\Domain\Exception;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class InvalidEntityClassException extends \LogicException implements DomainExceptionInterface
+{
+    public static function create(string $class): self
+    {
+        return new self(sprintf('Invalid entity class "%s" detected.', $class));
```

Comment:
While I always do this as well, it might be worth considering if `"Invalid entity class \"$class\" detected."` might be better.

https://3v4l.org/6bivC/vld#output vs https://3v4l.org/a7vqE/vld#output

Reply:
see #35, keeping `sprintf` globally for now